### PR TITLE
set-up mdn dna 2020 banner

### LINKED ISSUE #7492: Soapbox banner required for 2020 MDN Web DNA Survey
# Content Issue

- [ ] Please close this issue, I accidentally submitted it without adding any details
- [ ] New documentation
- [x] Correction or update

# URL of page you are seeing the problem on:

All MDN pages!

# Details

We want to launch the 2020 MDN Web DNA Survey on September 18th, and to coincide with it, we want to put a soapbox banner on MDN, inviting people to fill it out.

Last year's call to action text was:

> We have an opportunity for you to influence how browser vendors like Mozilla, Google, Microsoft, and Samsung prioritize feature development.

> It’s easy to take part, just complete this survey that was designed in collaboration with those browser vendors and the W3C. It will take you approximately 20 minutes — this is your chance to make yourself heard. The results and learnings will be shared publicly on MDN.

> You must be 18 years old or over to participate. We handle your information as described in the Mozilla Privacy Policy.

The actual banner text will probably be similar to last year's, something like:

> MDN Web DNA
> Help us understand the top 10 needs of web designers and developers.
> Take the survey.

Let us know what else you need for this. Thanks!

### PR DESCRIPTION
**Pull request by schalkneethling**
Add MDN Web DNA survey banner for 2020.
This is behind the flag developer_needs

NOTE: This still uses last year's survey link. It is behind a flag so, we could merge an update the link once we know the new one.

Screenshot 2020-09-10 at 15 06 58

Part of #7492

### CODE REVIEW

**Comment by schalkneethling** [comment]
Huh, that is weird. I ran `docker-compose exec web make black` and it seemed happy. Guess I need to pull the latest Dockers ;)

**Review by peterbe — CHANGES_REQUESTED** [review]
This is my passive aggressive way of reminding you to merging https://github.com/mdn/kuma/pull/7495 and then merging `master` into this branch here. :)
Also, a good way to remember where I was.

File: docs/feature-toggles.rst [hunk 1]
```diff
@@ -39,7 +39,7 @@ other advanced criteria.
 
 * ``contrib_beta`` - Enable/disable the contributions popup and pages
 * ``kumaediting`` - Enable/disable wiki editing.
-* ``user_interviews`` - Enable/disable user interviews invitation survey banner.
+* ``developer_needs`` - Enable/disable the MDN developer needs survey banner
```
Comment (peterbe) [inline docs/feature-toggles.rst:42]: You can delete this line now. `developer_needs` is already mentioned above where it belongs.

Comment (schalkneethling) [reply docs/feature-toggles.rst]: Done!

**Review by peterbe — APPROVED** [review]

**Pull request merged**